ansible/roles/test/tasks/test_sonic_by_testname.yml

```diff
@@ -0,0 +1,38 @@
+- debug: msg="!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!"
+- debug: msg="!!!!!!!!!!!!!!!!!!!! start to run test {{ testcase_name }} !!!!!!!!!!!!!!!!!!!!"
+- debug: msg="!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!"
+
+- name: do basic sanity check before each test
+  include: base_sanity.yml
+
+- name: validate all interfaces is up
+  include: interface.yml
+
+########### if your test playbook requires more extra_vars than ptf_host and testbed_type, you may specify them here
+########### ptf_host and testbed_type are handled by default
+########### configure extra vars if your testcases need more vars
+########### or when you call the playbook, you have to specify your extra_vars
```

Comment:
Agree. I was trying to see if we could call all tests by topology, but ansible playbook is not a good way to do it since then loop for test name and other look logic within test cases are conflicted. so left it only work for call single test case.  
To have a quick way to call each test without tag, then will need to change/re-wrap most of the test_case.yml file. 
I can either do it in 2 phases to improve it later, or abandon this one and make it more thorogh